lib/matplotlib/collections.py

```diff
@@ -1254,15 +1300,21 @@ class AsteriskPolygonCollection(RegularPolyCollection):
 
 
 class LineCollection(Collection):
-    """
-    All parameters must be sequences or scalars; if scalars, they will
-    be converted to sequences.  The property of the ith line
-    segment is::
+    r"""
+    Represents a sequence of `~.lines.Line2D`\s that should be drawn together.
 
-       prop[i % len(props)]
+    This class extends `.Collection` to represent a sequence of
+    `~.lines.Line2D`\s instead of just a sequence of `~.patches.Patch`\s.
+    Just as in `.Collection`, each property of a *LineCollection* may be either
+    a single value or a list of values. This list is then used cyclically for
+    each element of the LineCollection, so the property of the ``i``\th element
+    of the collection is::
 
-    i.e., the properties cycle if the ``len`` of props is less than the
-    number of segments.
+      prop[i % len(prop)]
+
+    A *LineCollection*'s properties default to their ``'lines'`` values in
+    `~.rcParams` instead of their ``'patch'`` values, and the
+    property *colors* is added in place of *edgecolors*.
```

Comment:
```suggestion
    The *LineCollection* properties default to :rc:`lines.*` instead of
    :rc:`patch.*`, and the property *colors* is added in place of
    *edgecolors*.
```